### PR TITLE
feature/32067/validation

### PR DESCRIPTION
**Pull request by massimez**

### CODE REVIEW

File: pages/profile.vue [hunk 1]
```diff
@@ -447,12 +451,14 @@ export default {
         model: localUserData.firstName,
         selector: 'FIRST-NAME',
         placeholder: firstName,
+        rules: 'required||max:15||alpha_spaces_dash',
```
Comment (matykoks) [inline pages/profile.vue:454]: why double ||

File: store/user/actions.js [hunk 2]
```diff
@@ -116,9 +116,9 @@ export default {
   },
   async editProfile({ commit }, { config, method }) {
     try {
-      const { result, ok } = await this.$axios.$put(method, config);
+      const result = await this.$axios.$put(method, config);
       commit('setUserData', result);
-      return ok;
+      return result;
```
Comment (matykoks) [inline store/user/actions.js:121]: for what are you changed it?

File: pages/profile.vue (hunk 1 above)
Comment (massimez) [reply pages/profile.vue]: Typo

File: store/user/actions.js (hunk 2 above)
Comment (massimez) [reply store/user/actions.js:121]: Adapting that to the endpoint response

**Pull request merged**